tracer/test/Datadog.Trace.Tests/Logging/DirectSubmission/DirectLogSubmissionSettingsTests.cs

```diff
@@ -210,13 +210,13 @@ public void ApiKey(string value, string expected)
         }
 
         [Theory]
-        [MemberData(nameof(BooleanTestCases), false)]
+        [MemberData(nameof(BooleanTestCases), true)]
```

Comment:
Now that we have the setting back in the `TracerSettings` class, can you move this entire method to the `tracer/test/Datadog.Trace.Tests/Configuration/TracerSettingsTests.cs` file?